example-jobs/checkmarx_security_job.groovy

```diff
@@ -1,16 +1,15 @@
 import jenkins.automation.builders.CheckmarxSecurityJobBuilder
 
 def projectName ='foo'
-def groupId = "your-group-id"
+def groupId = "ac43cb0d-034d-4b1e-9bf5-e7c1c46f71d2"
```

Comment:
NB: I am including this real-looking-guid-string on purpose so as to make it clearer what sort of value will be used for `groupId` in practice.